src/components/card-with-image/card-with-image.styles.js

```diff
@@ -0,0 +1,52 @@
+export const styles = {
+  left: {
+    box: {
+      display: 'flex',
+      width:'1128px',

```

Comment:
If we will need this component with full width? Components should not have fixed sizes.

Reply:
Create container, and move this component code to container